sbi/neural_nets/density_estimators/zuko_flow.py

```diff
@@ -141,7 +143,13 @@ def sample_and_log_prob(
 
         emb_cond = self._embedding_net(condition)
         dists = self.net(emb_cond)
-        samples, log_probs = dists.rsample_and_log_prob(sample_shape)
+
+        if isinstance(self.net, zuko.flows.GMM):
```

Comment:
Hello @anastasiakrouglova, a GMM is a density estimator, but not a normalizing flow (there is no `transformation`). You might want to implement that somewhere else.